src/main/java/com/indeed/jiraactions/jiraissues/JiraIssuesProcess.java

```diff
@@ -0,0 +1,139 @@
+        // Changes the issue from a String[] to a Map<String, String>
+        for(int i = 0; i < issue.length; i++) {
+            mappedLine.put(oldFields.get(i), issue[i]);
+        }
+        // Filters issues to a maximum of a year ago (for unclosed issues) and 6 months (for closed issues).
+        if(mappedLine.containsKey("lastupdated")) {
+            if(Integer.parseInt(mappedLine.get("lastupdated")) < filter) {
+                return null;
+            }
+        }
+
+        for(Map<String, String> updatedIssue: newIssuesMapped) {
+            if (mappedLine.get("issuekey").equals(updatedIssue.get("issuekey"))) {
+                newIssuesMapped.remove(updatedIssue);
+                return updatedIssue;  // Replace
+            }
+        }
+            return updateIssue(mappedLine);   // Update
+    }
+
+    public List<Map<String, String>> getRemainingIssues() {
+        final List<Map<String, String>> addedIssues = new ArrayList<>();
+        if (!newIssues.isEmpty()) {
+            addedIssues.addAll(newIssuesMapped);
+            log.debug("Added {} new issues.", addedIssues.size());
+        }
+        return addedIssues;
+    }
+
+    public Map<String, String> updateIssue(final Map<String, String> mappedLine) {
+        final long DAY = TimeUnit.DAYS.toSeconds(1);
+        final String status = formatStatus(mappedLine.get("status"));
+        try {
+            mappedLine.replace("issueage", String.valueOf(Long.parseLong(mappedLine.get("issueage")) + DAY));
+            mappedLine.replace("time", String.valueOf(Long.parseLong(mappedLine.get("time")) + DAY));
+            if(!mappedLine.containsKey("totaltime_" + status)) {
+                nonApiStatuses.add(mappedLine.get("status"));
+            } else {
+                mappedLine.replace("totaltime_" + status, mappedLine.get("totaltime_" + status), String.valueOf(Long.parseLong(mappedLine.get("totaltime_" + status)) + DAY));
+            }
+        } catch (final NumberFormatException e) {
+            log.error("Value of field is not numeric.", e);
+        }
+
+        // This part is very important in making sure that the previous TSV will conform to the new fields
+        final Map<String, String> mappedLineNewFields = new LinkedHashMap<>();
+        for(String field : newFields) {
+            if(!mappedLine.containsKey(field)) {
+                if(field.startsWith("totaltime") || field.startsWith("timetofirst") || field.startsWith("timetolast")) {
+                    mappedLineNewFields.put(field, "0");
+                } else {
+                    mappedLineNewFields.put(field, "");
+                }
+            } else {
+                mappedLineNewFields.put(field, mappedLine.get(field));
+            }
+        }
+        return mappedLineNewFields;
+    }
+
+    private String formatStatus(final String status) {
+        if (status.equals("")) {
+            return "";
+        }
+        return status
```

Comment:
This is duplicated code from somewhere else; seems like a good candidate to consolidate.